linkis-computation-governance/linkis-engineconn/linkis-computation-engineconn/src/main/scala/org/apache/linkis/engineconn/computation/executor/execute/ConcurrentComputationExecutor.scala

```diff
@@ -26,11 +26,46 @@ abstract class ConcurrentComputationExecutor(override val outputPrintLimit: Int
     extends ComputationExecutor(outputPrintLimit)
     with ConcurrentExecutor {
 
+  private val EXECUTOR_STATUS_LOCKER = new Object
+
+  override def execute(engineConnTask: EngineConnTask): ExecuteResponse = {
+    if (isBusy) {
+      logger.error(
+        s"Executor is busy but still got new task ! Running task num : ${getRunningTask}"
+      )
+    }
+    if (getRunningTask >= getConcurrentLimit) EXECUTOR_STATUS_LOCKER.synchronized {
+      if (getRunningTask >= getConcurrentLimit && NodeStatus.isIdle(getStatus)) {
+        logger.info(
+          s"running task($getRunningTask) > concurrent limit $getConcurrentLimit, now to mark engine to busy "
```

Comment:
`running task: $getRunningTask > concurrent limit: $getConcurrentLimit, now to mark engine to busy`
or `running task($getRunningTask) > concurrent limit($getConcurrentLimit), now to mark engine to busy`?